src/write.c

```diff
@@ -941,6 +941,30 @@ static avifCodecType avifEncoderGetCodecType(const avifEncoder * encoder)
     return avifCodecTypeFromChoice(encoder->codecChoice, AVIF_CODEC_FLAG_CAN_ENCODE);
 }
 
+static avifBool avifEncoderDataForceKeyframeForAlpha(const avifEncoderData * data, const avifEncoderItem * colorItem, avifAddImageFlags addImageFlags)
+{
+    if (!data->alphaPresent) {
+        // There is no alpha plane.
+        return AVIF_FALSE;
+    }
+    if (addImageFlags & AVIF_ADD_IMAGE_FLAG_SINGLE) {
+        // Not an animated image.
+        return AVIF_FALSE;
+    }
+    if (data->frames.count == 0) {
```

Comment:
Nit: Document that `data->frames.count` is the number of frames that have been encoded by previous `avifEncoderAddImage()` calls. This fact is important to understanding this function, but is not obvious unless I inspect the caller's code.

Reply:
Done.